drivers/volume/portworx/portworx.go

```diff
@@ -559,11 +559,21 @@ func (d *portworx) WaitForUpgrade(n node.Node, newVersion string) error {
 			}
 		}
 
+		// filter out first 3 octets from the tag
+		matches := regexp.MustCompile(`^(\d+\.\d+\.\d+)-.*`).FindStringSubmatch(tag)
```

Comment:
this may not work for portworx releases like 1.2.11.4.. you may want an additional \.d* after the first group

Reply:
Removed the - in new changes which will handle 1.2.11.4. Note that we only match the first 3 octets.